### ISSUE TITLE
Vulns within clair images

### ISSUE THREAD

**Issue by ErikThoreson** [quay/clair#322]
Are there plans to remediate the vulnerabilities within the clair images on quay.io? If not, why not?

This is the link to the current pre-release tagged version:
https://quay.io/repository/coreos/clair/image/b48038b10af3e97cb0eca8ce80ee6646c8dd0622cd4b9703634f06b779cf8f29?tab=vulnerabilities

**Comment by yfoelling** [comment]
@ErikThoreson The Base-Image clair uses is full of Vulnerabilities, i made one myself with centos base.
This will work much better.

**Comment by ErikThoreson** [comment]
@yfoelling that seems prudent given the current state of the images.  However, it begs the question, if scanning for vulns is important to coreos why not remediate their own?

**Comment by Quentin-M** [comment]
Thanks for pointing out this particular issue.

We do give an important attention to the state of our images. One of the most obvious example would be hyperkube, which we patched several times, within hours, in order to fix various vulnerabilities (e.g. jq).

None of the vulnerabilities seemed to affect Clair directly. However, I just took the opportunity to make Clair use Alpine 3.5 by default (https://github.com/coreos/clair/pull/325). The squashed image is now 41MB/113MB big and has way less fluff packaged.
